server/services/nl.py

```diff
@@ -58,31 +60,32 @@ def _download_embeddings(self):
       _, filename = os.path.split(blob.name)
       blob.download_to_filename(os.path.join(TEMP_DIR, filename))  # Download
 
-  def detect_svs(self, query):
+  def detect_svs(self, query, embeddings_build):
     query_embeddings = self.model.encode([query])
+    if embeddings_build not in self.dataset_embeddings_maps:
+      return {'SV': [], 'CosineScore': []}
```

Comment:
nit: raise ValueError('Invalid embedding build value')? Then catch it from caller and record it in the debug dict?

Reply:
Thanks for the suggestion! Done :)